src/head.c

```diff
@@ -154,8 +154,22 @@ wl_fixed_t head_auto_scale(struct Head *head) {
 	// round the dpi to the nearest 12, so that we get a nice even wl_fixed_t
 	long dpi_quantized = (long)(dpi / 12 + 0.5) * 12;
 
+	// convert min and max to quantized dpi
+	long dpi_min = 12 * (long)(min * 8 + 0.9999f);
```

Comment:
Why the 0.9999f ? Is it to do with the double parameters? Could they be floats?

Reply:
This was a quick-n-dirty implementation of `ceil`, ie. the value needs to be rounded up. I'll write something smarter instead.

Reply:
Changed to ceil() from math.h.